variables.tf

```
@@ -167,6 +167,18 @@ variable "artifact_location" {
   description = "Location of artifact. Applies only for artifact of type S3"
 }
 
+variable "artifact_encryption_disabled" {
+  type        = bool
+  default     = false
```

Cloud Posse standard naming is to use `enabled` rather than `disabled`

```suggestion
variable "artifact_encryption_enabled" {
  type        = bool
  default     = true
```